openbsd/OpenBSDProcessList.c

```diff
@@ -76,6 +86,21 @@ ProcessList* ProcessList_new(UsersTable* usersTable, Hashtable* pidMatchList, ui
 
    opl->cpuSpeed = -1;
 
+   size = sizeof(cpu_stats);
+   for (unsigned int i = 0; i < ncpu; i++) {
+      ncmib[2] = i;
```

Comment:
Move actual initialization of `ncmib` here and make it const.
```suggestion
      const int ncmib[] = { CTL_KERN, KERN_CPUSTATS, i };
```